tests/integration/targets/ejabberd_user/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Bail out if not supported
+  ansible.builtin.meta: end_play
+  when: ansible_distribution in ('Alpine', 'openSUSE Leap', 'CentOS', 'Fedora')
+
+- name: Create user without ejabberdctl installed
+  community.general.ejabberd_user:
+    host: localhost
+    username: alice
+    password: pa$$w0rd
+    state: present
+  register: user_no_ejabberdctl
+  ignore_errors: true
+
+- name: Install ejabberd
+  ansible.builtin.package:
+    name: ejabberd
+    state: present
+  notify: Remove ejabberd
```

Comment:
If the package is already installed (for whatever reason), the above check (without ejabberdctl installed) will fail. How about registering the result of this task and change the check for `user_no_ejabberdctl` so that it only tests `user_no_ejabberdctl` if this task is changed?

Reply:
Or we can forcefully remove the package before that.